src/promptflow-evals/promptflow/evals/synthetic/adversarial_simulator.py

```diff
@@ -108,39 +111,44 @@ async def __call__(
         Executes the adversarial simulation against a specified target function asynchronously.
 
         :param scenario: Enum value specifying the adversarial scenario used for generating inputs.
+                         example:
+
+                         * AdversarialScenario.ADVERSARIAL_QA
+                         * AdversarialScenario.ADVERSARIAL_CONVERSATION
         :type scenario: AdversarialScenario

```

Comment:
I guess the doc CI raised warning because of this line, `more than one target found for cross-reference 'AdversarialScenario': promptflow.evals.synthetic.AdversarialScenario, promptflow.evals.synthetic.adversarial_scenario.AdversarialScenario`, maybe specify the full path of this class to fix this issue.

Reply:
Fixed and made this a link.